From bd72508eaf30243d5d15d65a3c7fbeb59b1f175b Mon Sep 17 00:00:00 2001
From: Michael Haberler <git@mah.priv.at>
Date: Mon, 31 Mar 2014 14:15:24 +0200
Subject: [PATCH] arm: remove reference to mach/timex.h in
 arch/arm/include/asm/timex.h

looks like a leftover - the other references to mach/timex.h already
have been removed as outline in http://www.spinics.net/lists/arm-kernel/msg291792.html
---
 arch/arm/include/asm/timex.h |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm/include/asm/timex.h b/arch/arm/include/asm/timex.h
index 83f2aa8..f6fcc67 100644
--- a/arch/arm/include/asm/timex.h
+++ b/arch/arm/include/asm/timex.h
@@ -12,12 +12,6 @@
 #ifndef _ASMARM_TIMEX_H
 #define _ASMARM_TIMEX_H
 
-#ifdef CONFIG_ARCH_MULTIPLATFORM
-#define CLOCK_TICK_RATE 1000000
-#else
-#include <mach/timex.h>
-#endif
-
 typedef unsigned long cycles_t;
 #define get_cycles()	({ cycles_t c; read_current_timer(&c) ? 0 : c; })
 
-- 
1.7.10.4

